packages/ua-blocker/tsconfig.script.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "tsBuildInfoFile": "dist/tsconfig.script.tsbuildinfo",
+    "allowImportingTsExtensions": true,
+    "noEmit": true,
+    "types": ["node"]
+  },
+  "include": ["**/*.ts"],
+  "exclude": ["**/*.test.ts"],
+  "references": []
+}
```

Comment:
I also moved the `get-robots-txt.ts` from `./src/script` to a top level `./script` directory. This just allows a seperate tsconfig for the script, middleware, and tests files